CHANGELOG.md

```diff
@@ -1,5 +1,37 @@
 # Changelog
 
+## 2.51.0-rc.0 / 2024-03-07
+
+This version is built with Go 1.22.1.
+
+There is a new optional build tag "dedupelabels", which should reduce memory consumption (#12304).
+It is off by default; there will be an optional alternative image to try it out.
+
+* [BUGFIX] PromQL: Constrain extrapolation in rate() to half of sample interval #13725
```

Comment:
Sorry this is not supposed to be at the top; I will move it down into the BUGFIX section.